src/main/java/com/meraphorce/controllers/UserController.java

```diff
@@ -16,12 +25,120 @@ public class UserController {
+    				.message("Error to create user.")
+    				.status(false)
+    				.build();
+    	}
+    	
+    	response = ResponseEntity.ok(responseDTO);
+    	
+        return response;
     }
 
     @GetMapping
-    public ResponseEntity<List<User>> getAllUsers(){
-        return ResponseEntity.ok(userService.getAllUsers());
+    public ResponseEntity<ResponseDTO<?>> getAllUsers() {
+    	ResponseEntity<ResponseDTO<?>> response = null;
+    	ResponseDTO<?> responseDTO = null;
+    	List<UserDTO> listUsers = null;
+    	
+    	listUsers = this.userService.getAllUsers();
+    	
+    	if(listUsers != null) {
+    		responseDTO = ResponseDTO.builder()
+    				.object(listUsers)
+    				.status(true)
+    				.build();
+    	} else {
+    		responseDTO = ResponseDTO.builder()
+    				.message("Error to get all users.")
+    				.status(false)
+    				.build();
+    	}
+    	
+    	response = ResponseEntity.ok(responseDTO);
+    	
+        return response;
+    }
+    
+    @GetMapping("/{id}")
+    public ResponseEntity<ResponseDTO<?>> getUser(@PathVariable String id) {
+    	ResponseEntity<ResponseDTO<?>> response = null;
+    	ResponseDTO<?> responseDTO = null;
+    	UserDTO userData = null;
+    	
+    	userData = this.userService.getUserById(id);
+    	
+    	if(userData != null) {
+    		responseDTO = ResponseDTO.builder()
+    				.object(userData)
+    				.status(true)
+    				.build();
+    	} else {
+    		responseDTO = ResponseDTO.builder()
+    				.message("Error to get User[" + id + "].")
+    				.status(true)
+    				.build();
+    	}
+    	
+    	response = ResponseEntity.ok(responseDTO);
+    	
+        return response; 
+    }
+    
+    @PutMapping("/{id}")
+    public ResponseEntity<ResponseDTO<?>> updateUser(@PathVariable String id, @RequestBody UserDTO user) {
+    	ResponseEntity<ResponseDTO<?>> response = null;
+    	ResponseDTO<?> responseDTO = null;
+    	UserDTO updateUserData = null;
+    	
+    	updateUserData = this.userService.updateUser(id, user);
+    	
+    	if(updateUserData != null) {
+    		responseDTO = ResponseDTO.builder()
+    				.object(updateUserData)
+    				.status(true)
+    				.build();
+    	} else {
+    		responseDTO = ResponseDTO.builder()
+    				.message("Error to update User[" + id + "].")
+    				.status(true)
+    				.build();
+    	}
+    	
+    	response = ResponseEntity.ok(responseDTO);
+    	
+    	return response;
+    }
+    
+    @DeleteMapping("/{id}")
+    public ResponseEntity<ResponseDTO<?>> deleteUser(@PathVariable String id) {
+    	ResponseEntity<ResponseDTO<?>> response = null;
+    	ResponseDTO<?> responseDTO = null;
+    	boolean  isDeleted = false; 
+    	
+    	isDeleted = this.userService.deleteUser(id);
+    	
+    	responseDTO = ResponseDTO.builder()
+				.message(isDeleted? "User deleted": "Error to delete User[" + id + "].")
+				.status(isDeleted)
+				.build();
+    	
+    	response = ResponseEntity.ok(responseDTO);
+    	
+    	return response;
```

Comment:
Add new line.
Ending a file with a newline (also known as a line break or an end-of-line (EOL) character) is a convention rooted in the definitions of Unix-based systems and is considered a best practice for several practical reasons in software development